### ISSUE TITLE
Standard Industry Classifications?

### ISSUE THREAD

**Issue by BobHarper1** [SocialInvestmentDataLab/spec#23]
[SIC codelist](https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/527619/SIC07_CH_condensed_list_en.csv/preview)

Is the SIC a useful, albeit fairly blunt, organisation classification to include in the Spec? The Spec will _possibly_ mandate codelists for activity types by investee organisations. Given that the SIC is generalist and not focused on the social sector it might not be an appropriate classifier (round peg, square hole).

As registered companies have SICs listed and available from Companies House, is it necessary to include SICs in the Spec where these may or may not differ for organisations with known company numbers?

A report (which focuses on lending by mainstream banks to SEs and charities) http://flipfinance.org.uk/2016/07/23/forest-for-the-trees/ takes an interesting analytical approach of using SICs to attempt to identify what are social sector organisations, but summarises the limits of such an approach:
> This is therefore a classification, though not of ownership, governance or legal form, nor of motivation, approach to impact or profit distribution. For this reason, it is impossible to use SIC codes to accurately assess activity relating uniquely to the social sector (however defined). _(p15)_

**Comment by ScatteredInk** [comment]
I don't think we want SICs to have a privileged position in terms of classifications, and we also need to deal with the fact that there are multiple SICs (in terms of number of digits and different revisions by year. There is also the edge case of foreign-registered charities active in the UK. So, essentially, SICs need to be treated like any other classification. 

But I definitely think that there are enough use-cases and that SICs are confusing enough - especially when dealing with definitional breaks in longitudinal analysis - that we should plan for significant guidance around _using_ SICs in the spec and for the possibility that SICs will be used in different ways than they are in dealings with Companies House.

I agree that there are drawbacks from SICs not being sector-specific but, given the emphasis on places in the impact-measurement use-cases, the intersection with other statistics on the labour market and business demography makes this a relatively minor point - and if something better comes along, then the techniques are in place to switch over.
